src/Bravo3/ImageManager/Services/ImageManager.php

```diff
@@ -603,7 +607,14 @@ public function addEncoder(EncoderInterface $encoder, $prepend = false)
     public function rename($source_key, $target_key)
     {
         $this->filesystem->rename($source_key, $target_key);
-        
+
+        if ($this->tagExists($source_key)) {
+            $metadata = $this->getMetadata($source_key);
+
+            $this->tag($target_key, $metadata);
```

Comment:
You've raised an interesting point here, if the source tag never existed then we never create a target tag. Yet it's possible that the source _file_ existed, but it's tag did not. If the file doesn't exist, then should filesystem->rename fail? If so, then we should always set the target key.